bundles/org.openhab.binding.roku/README.md

```diff
@@ -109,17 +109,19 @@ roku:roku_tv:mytv1 "My Roku TV" [ hostName="192.168.10.1", refresh=10 ]
 
 String Player_ActiveApp        "Current App: [%s]"         { channel="roku:roku_player:myplayer1:activeApp" }
 String Player_ActiveAppName    "Current App Name: [%s]"    { channel="roku:roku_player:myplayer1:activeAppName" }
-String Player_Button           "Send Command to Roku"      { channel="roku:roku_player:myplayer1:button" }
+String Player_Button           "Send Command to Roku"      { channel="roku:roku_player:myplayer1:button", autoupdate="false" }
```

Comment:
Seeing this reminded me of #12141. Do you think a similar solution would be relevant here as well? Not for this PR obviously, just a side note.

Reply:
Is that just a substitute for  

> Seeing this reminded me of #12141. Do you think a similar solution would be relevant here as well? Not for this PR obviously, just a side note.

I can address that in a future PR. Is this just a substitute/implicit way of setting autoupdate="false"?

Reply:
The **veto** policy will make sure that the command is sent, but without automatically updating the item state:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#auto-update-policies